config.go

```diff
@@ -181,6 +181,8 @@ type config struct {
 	BitcoindMode *bitcoindConfig `group:"bitcoind" namespace:"bitcoind"`
 	NeutrinoMode *neutrinoConfig `group:"neutrino" namespace:"neutrino"`
 
+	Bcash *chainConfig `group:"Bcash" namespace:"bcash"`
```

Comment:
Should the casing here be `group:"BCash"`?  We should reach out to the bcash core team and understand which is the proper casing.